### ISSUE TITLE
Just a suggestion? idk '-' 

### ISSUE THREAD

**Issue by Yurkhun** [Nem4/viewerbot#76]
It looks like a script that sends HTTP requests to a Twitch channel using a list of proxies and a random user agent. The requests are sent concurrently using multiple threads.

There are a few issues with this code that could potentially cause problems:

1. The script uses the **fake_useragent** module to generate random user agents for each request, but this module has been deprecated and is no longer maintained. You should consider using a different method to generate random user agents.

2. The script uses the **Streamlink** module to fetch the stream URL for the channel, but it does not handle exceptions when the stream is not available. This could cause the script to crash if the stream is not found.

3. The script uses a fixed number of threads (**max_nb_of_threads**) to send requests concurrently, which could cause problems if the number of threads is too high. You should consider using a thread pool or limiting the number of threads based on system resources.

4. The script uses a hardcoded list of good proxies, but it does not validate the proxies or check if they are still working. This could cause issues if some of the proxies are no longer functional or if they are blocked by the server.

Overall, the code has some basic issues that need to be addressed before it can be considered a good working code.

**Comment by Anonymous2** [comment]
Indeed, this doesn't work for some time now. We need a project manager probably, or at least someone with knowledge and some time to code to bring the project back from the grave.
